schema/verify/appschema.sql

```diff
@@ -0,0 +1,10 @@
+-- Verify todo_app:appschema on pg
+
+BEGIN;
+
+DO $$
+BEGIN
+    ASSERT (SELECT has_schema_privilege('todo_app', 'usage'));
+END $$;
```

Comment:
```suggestion
select has_schema_privilege('todo_app', 'usage');
```
this would work as well